client/src/components/messenger/ContactItem.js

```diff
@@ -70,8 +70,7 @@ const ContactItem = ({ img, username, status, id, triggerRefresh }) => {
     };
 
     return (
-        <div className="contact_item">
-
+        <div className="contact_item" title={username + "#" + id}>
```

Comment:
lines 12, 26, 75, 76 should be updated to include the id
lines 19, 33, 79 should be updated to include # and the id